packages/block-editor/src/components/provider/index.js

```diff
@@ -121,11 +121,9 @@ class BlockEditorProvider extends Component {
 		const { children } = this.props;
 
 		return (
-			<SlotFillProvider>
-				<DropZoneProvider>
-					{ children }
-				</DropZoneProvider>
-			</SlotFillProvider>
+			<DropZoneProvider>
```

Comment:
Makes me wonder about this as well :) It feels like it's the same questions could be asked for this provider.

Reply:
> Makes me wonder about this as well :) It feels like it's the same questions could be asked for this provider.

Probably. Let's sort it out separately?